stripe/src/main/java/com/stripe/android/model/SourceOwner.java

```diff
@@ -0,0 +1,221 @@
+    private static final String FIELD_VERIFIED_NAME = VERIFIED + FIELD_NAME;
+    private static final String FIELD_VERIFIED_PHONE = VERIFIED + FIELD_PHONE;
+
+    private SourceAddress mAddress;
+    private String mEmail;
+    private String mName;
+    private String mPhone;
+    private SourceAddress mVerifiedAddress;
+    private String mVerifiedEmail;
+    private String mVerifiedName;
+    private String mVerifiedPhone;
+
+    SourceOwner(
+            SourceAddress address,
+            String email,
+            String name,
+            String phone,
+            SourceAddress verifiedAddress,
+            String verifiedEmail,
+            String verifiedName,
+            String verifiedPhone) {
+        mAddress = address;
+        mEmail = email;
+        mName = name;
+        mPhone = phone;
+        mVerifiedAddress = verifiedAddress;
+        mVerifiedEmail = verifiedEmail;
+        mVerifiedName = verifiedName;
+        mVerifiedPhone = verifiedPhone;
+    }
+
+    SourceAddress getAddress() {
+        return mAddress;
+    }
+
+    void setAddress(SourceAddress address) {
+        mAddress = address;
+    }
+
+    String getEmail() {
+        return mEmail;
+    }
+
+    void setEmail(String email) {
+        mEmail = email;
+    }
+
+    String getName() {
+        return mName;
+    }
+
+    void setName(String name) {
+        mName = name;
+    }
+
+    String getPhone() {
+        return mPhone;
+    }
+
+    void setPhone(String phone) {
+        mPhone = phone;
+    }
+
+    SourceAddress getVerifiedAddress() {
+        return mVerifiedAddress;
+    }
+
+    void setVerifiedAddress(SourceAddress verifiedAddress) {
+        mVerifiedAddress = verifiedAddress;
+    }
+
+    String getVerifiedEmail() {
+        return mVerifiedEmail;
+    }
+
+    void setVerifiedEmail(String verifiedEmail) {
+        mVerifiedEmail = verifiedEmail;
+    }
+
+    String getVerifiedName() {
+        return mVerifiedName;
+    }
+
+    void setVerifiedName(String verifiedName) {
+        mVerifiedName = verifiedName;
+    }
+
+    String getVerifiedPhone() {
+        return mVerifiedPhone;
+    }
+
+    void setVerifiedPhone(String verifiedPhone) {
+        mVerifiedPhone = verifiedPhone;
+    }
+
+    @NonNull
+    @Override
+    public Map<String, Object> toMap() {
+        Map<String, Object> hashMap = new HashMap<>();
+        if (mAddress != null) {
+            hashMap.put(FIELD_ADDRESS, mAddress.toMap());
+        }
+        hashMap.put(FIELD_EMAIL, mEmail);
+        hashMap.put(FIELD_NAME, mName);
+        hashMap.put(FIELD_PHONE, mPhone);
+        if (mVerifiedAddress != null) {
+            hashMap.put(FIELD_VERIFIED_ADDRESS, mVerifiedAddress.toMap());
+        }
+        hashMap.put(FIELD_VERIFIED_EMAIL, mVerifiedEmail);
+        hashMap.put(FIELD_VERIFIED_NAME, mVerifiedName);
+        hashMap.put(FIELD_VERIFIED_PHONE, mVerifiedPhone);
+        StripeNetworkUtils.removeNullParams(hashMap);
```

Comment:
Could you make this a static import like the other file: `SourceRedirect` or the other way round, for consistency?